python_transport/wirepas_gateway/protocol/mqtt_wrapper.py

```diff
@@ -206,6 +223,14 @@ def run(self):
             # thread has exited
             self.on_termination_cb()
 
+    def _publish_from_main_thread(self, topic, payload, qos, retain):
+        mid = self._client.publish(topic, payload, qos=qos, retain=retain).mid
+        # self.logger.info("Publish {} qos={}".format(mid, qos))
+        if self.publish_gueue_size == 0:
```

Comment:
Typo in attribute name, it should read self.publish_queue_size

Reply:
Done